### DISCUSSION TITLE
Add model display widget for server-editor module

### DISCUSSION THREAD

**Discussion by AbbottMc** [Mojang/minecraft-editor#183] [Feedback]
If possible, can you guys add the block/entity model display widget for editor module so that we can display some model for our need (such as display model when picking entity or block) (dreaming time! (XD) )

**Comment by m-brisch** [comment]
Thanks for the feedback! The scenario is having them inside a pane (similar to what we have for the block picker when customizing the hotbar) or do you mean an in-game visualization?

**Comment by AbbottMc** [comment]
> Thanks for the feedback! The scenario is having them inside a pane (similar to what we have for the block picker when customizing the hotbar) or do you mean an in-game visualization?

@m-brisch yeah the scenario I'm thinking is a widget that we can add into any ui panel/pane and we can specify the model type (such as the block or entity it shows), or specify the model identifier directly.
And it would be great that if it can specify the animation that the model is playing
And I'm not quite sure what means "in-game visualization"? (
